scripts/cluster/start_cluster.sh

```
@@ -0,0 +1,193 @@
+        local raft_port=${NODE_RAFT_PORTS[$i]}
+        local node_num=$((i + 1))
+        
+        for j in {1..10}; do
+            if nc -z 127.0.0.1 $redis_port 2>/dev/null; then
+                log_success "Node $node_num Redis port $redis_port is ready"
+                break
+            fi
+            sleep 1
+        done
+        
+        for j in {1..10}; do
```

Shellcheck (0.11.0)

[warning] j appears unused. Verify use (or export if used externally).

(SC2034)